config/configgrpc/configgrpc.go

```diff
@@ -357,13 +358,13 @@ func (gss *GRPCServerSettings) ToServerOption(host component.Host, settings comp
 }
 
 // getGRPCCompressionName returns compression name registered in grpc.
-func getGRPCCompressionName(compressionType middleware.CompressionType) (string, error) {
+func getGRPCCompressionName(compressionType configcompression.CompressionType) (string, error) {
 	switch compressionType {
-	case middleware.CompressionGzip:
+	case configcompression.CompressionGzip:
```

Comment:
```suggestion
	case configcompression.Gzip:
```

If we stick with configcompression, remove "compression" from the property name.

Reply:
done. Thank you